### DISCUSSION TITLE
Soroban Illustration NFTs

### DISCUSSION THREAD

**Discussion by jaggernaut85** [stellar/sorobanathon#141] [General]
[https://litemint.com/items/GDGBXK5XIVYVYCI6NGZWO3YNVG5RLRQ42X4BPEXCVWROR53P7EK4PWAV/adventure](url)

[https://litemint.com/items/GD2G2KGMJSALUUKY2NUKKCDCZTHUKS4W7WMJRNPPRXL7LLQM6N3F2HPV/stonedoor](url)

[https://litemint.com/items/GAUXLPQ2J74JXNQQATADXSW4AU46WJ3GHGXEUT7Q4ISJZ6X25EBKXCJ4/lavasnakes](url)

[https://litemint.com/items/GCDTF5LVNNLXLZXEHLW2TVOGE2K4EXHTXOGGYCW5DOV3PF5TBDQZR4SL/creepylab](url)

[https://litemint.com/items/GAUF2PVNFLVNHALBZ3225CPS34O6YCFJGOUZOICGNHA7ZHBJUMCRK2KO/wiseoldman](url)

[https://litemint.com/items/GA3RTVS57L3JVZSZ6Z6DN44AXETQB4W7DOB4PYZTZ4QVUXKQVEUEGLS6/rotaryphone](url)

[https://litemint.com/items/GCRAW2KWI5ZO4PJ6S2X4FX3TOFWSTD7NGVGGVNVNAI3MWSDFXBRGYC4F/LuckFish](url)

**Comment by stellarjohn** [comment]
Hi [jaggernaut85](https://github.com/jaggernaut85)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
